### PR TITLE
storage: Disable "Apply" in Teardown dialogs after error

### PR DESCRIPTION
**Pull request by mvollmer**
We want the user to cancel and start over.

### CODE REVIEW

File: pkg/storaged/vdo-details.jsx [hunk 1]
```diff
@@ -138,6 +138,7 @@ export class VDODetails extends React.Component {
                     Teardown: TeardownMessage(usage),
                     Action: {
                         Title: _("Stop"),
+                        disable_on_error: usage.Teardown,
```
Comment (cockpituous) [inline pkg/storaged/vdo-details.jsx:141]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19579-20231107-135515-193b7257-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Review by jelly — APPROVED** [review]

**Review by jelly — CHANGES_REQUESTED** [review]
Unit tests fail:

# test/verify/check-storage-used:170:9: F841 Local variable `sleep_pid` is assigned to but never used

File: pkg/storaged/vdo-details.jsx (hunk 1 above)
Comment (cockpituous) [inline pkg/storaged/vdo-details.jsx:141]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19579-20231108-113532-872713b2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/vdo-details.jsx [hunk 2]
```diff
@@ -199,6 +200,7 @@
                 Action: {
                     Title: _("Delete"),
                     Danger: _("Deleting erases all data on a VDO device."),
+                    disable_on_error: usage.Teardown,
```
Comment (cockpituous) [inline pkg/storaged/vdo-details.jsx:203]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19579-20231108-113532-872713b2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Review by jelly — APPROVED** [review]

**Pull request merged**